flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -1,50 +1,132 @@
 from __future__ import absolute_import
 from __future__ import print_function
 
+import logging as _logging
+import math as _math
+import os as _os
+
 import click
 
 from flytekit.clis.sdk_in_container.constants import CTX_PACKAGES, CTX_PROJECT, CTX_DOMAIN, CTX_VERSION
-from flytekit.common import workflow as _workflow, utils as _utils
+from flytekit.common import utils as _utils
+from flytekit.common.core import identifier as _identifier
 from flytekit.common.exceptions.scopes import system_entry_point
 from flytekit.common.tasks import task as _sdk_task
 from flytekit.common.utils import write_proto_to_file as _write_proto_to_file
 from flytekit.configuration import TemporaryConfiguration
-from flytekit.configuration.internal import CONFIGURATION_PATH
-from flytekit.configuration.internal import IMAGE as _IMAGE
-from flytekit.models.workflow_closure import WorkflowClosure as _WorkflowClosure
+from flytekit.configuration import internal as _internal_configuration
 from flytekit.tools.module_loader import iterate_registerable_entities_in_order
 
 
 @system_entry_point
-def serialize_tasks(pkgs):
-    # Serialize all tasks
-    for m, k, t in iterate_registerable_entities_in_order(pkgs, include_entities={_sdk_task.SdkTask}):
-        fname = '{}.pb'.format(_utils.fqdn(m.__name__, k, entity_type=t.resource_type))
-        click.echo('Writing task {} to {}'.format(t.id, fname))
-        pb = t.to_flyte_idl()
-        _write_proto_to_file(pb, fname)
+def serialize_tasks_only(project, domain, pkgs, version, folder=None):
+    """
+    :param Text project:
+    :param Text domain:
+    :param list[Text] pkgs:
+    :param Text version:
+    :param Text folder:
+
+    :return:
+    """
+    # m = module (i.e. python file)
```

Comment:
could just use descriptive variable names in the first place to avoid these comments :P